File: iniciante/python/1134-tipo-de-combustivel.py
def tipo_de_combustivel():
    alcool = 0
    gasolina = 0
    diesel = 0
    while True:
        codigo_do_combustivel = int(input())
        if codigo_do_combustivel == 1:
            alcool += 1
        elif codigo_do_combustivel == 2:
            gasolina += 1
        elif codigo_do_combustivel == 3:
            diesel += 1
        elif codigo_do_combustivel == 4:
            break
    print(f'MUITO OBRIGADO\n'
          f'Alcool: {alcool}\n'
          f'Gasolina: {gasolina}\n'
          f'Diesel: {diesel}')


tipo_de_combustivel()

File: iniciante/c/1134-tipo-de-combustivel.c
#include <stdio.h>
#include <stdbool.h>

int main(){
    int escolha, alcool = 0, gasolina = 0, diesel = 0;
    while (true){
        scanf("%d", &escolha);
        if (escolha == 1){
            alcool++;
        }
        else if (escolha == 2){
            gasolina++;
        }
        else if (escolha == 3){
            diesel++;
        }
        else if (escolha == 4){
            break;
        }
    }
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", alcool);
    printf("Gasolina: %d\n", gasolina);
    printf("Diesel: %d\n", diesel);

    return 0;
}
